manager-bundle/src/ContaoManager/Plugin.php

```diff
@@ -334,25 +334,41 @@ private function addDefaultServerVersion(array $extensionConfigs, ContainerBuild
      *
      * @return array<string,array<string,array<string,array<string,mixed>>>>
      */
-    private function addDefaultPdoDriverOptions(array $extensionConfigs): array
+    private function addDefaultPdoDriverOptions(array $extensionConfigs, ContainerBuilder $container): array
     {
         // Do not add PDO options if the constant does not exist
         if (!\defined('PDO::MYSQL_ATTR_MULTI_STATEMENTS')) {
             return $extensionConfigs;
         }
 
+        $driver = null;
+        $url = null;
         foreach ($extensionConfigs as $extensionConfig) {
-            // Do not add PDO options if the selected driver is not pdo_mysql
-            if (isset($extensionConfig['dbal']['connections']['default']['driver']) && 'pdo_mysql' !== $extensionConfig['dbal']['connections']['default']['driver']) {
-                return $extensionConfigs;
+            if (isset($extensionConfig['dbal']['connections']['default']['driver'])) {
+                $driver = $extensionConfig['dbal']['connections']['default']['driver'];
+            }
+
+            if (isset($extensionConfig['dbal']['connections']['default']['url'])) {
+                $url = $container->resolveEnvPlaceholders($extensionConfig['dbal']['connections']['default']['url'], true);
```

Comment:
```suggestion
                $driver = parse_url($container->resolveEnvPlaceholders($extensionConfig['dbal']['connections']['default']['url'], true), PHP_URL_SCHEME);
```

We should IMO just extract the driver and check for all aliases of `pdo_mysql`:  `mysql`, `mysql2` and `pdo_mysql` isself.

Reply:
we _could_ use `parse_url` to check the driver a few lines below, but not here. Since both driver and URL _could_ be defined in multiple extension configs, and URL always wins over driver, we need to know which one to use here. Also, not sure if `parse_url` in the loop is efficient (compared to just extracting the scheme once below).